ERROR [2022-06-21 05:02:49,256] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (0/20) now.
ERROR [2022-06-21 05:02:50,289] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (1/20) in 1.00 second.
ERROR [2022-06-21 05:02:52,311] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (2/20) in 1.00 second.
ERROR [2022-06-21 05:02:54,334] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (3/20) in 1.00 second.
ERROR [2022-06-21 05:02:56,357] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (4/20) in 1.00 second.
ERROR [2022-06-21 05:02:58,381] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (5/20) in 1.00 second.
ERROR [2022-06-21 05:03:00,409] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (6/20) in 1.00 second.
ERROR [2022-06-21 05:03:02,440] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (7/20) in 1.00 second.
ERROR [2022-06-21 05:03:04,460] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (8/20) in 1.00 second.
ERROR [2022-06-21 05:03:06,489] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (9/20) in 1.00 second.
ERROR [2022-06-21 05:03:08,519] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (10/20) in 1.00 second.
ERROR [2022-06-21 05:03:11,587] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (11/20) in 1.00 second.
ERROR [2022-06-21 05:03:13,613] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (12/20) in 1.00 second.
ERROR [2022-06-21 05:03:15,641] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (13/20) in 1.00 second.
ERROR [2022-06-21 05:03:17,678] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (14/20) in 1.00 second.
ERROR [2022-06-21 05:03:19,711] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (15/20) in 1.00 second.
ERROR [2022-06-21 05:03:21,738] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (16/20) in 1.00 second.
ERROR [2022-06-21 05:03:23,766] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (17/20) in 1.00 second.
ERROR [2022-06-21 05:03:25,789] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (18/20) in 1.00 second.
ERROR [2022-06-21 05:03:27,817] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (19/20) in 1.00 second.
CRITICAL [2022-06-21 05:03:29,839] - [celery.backends.redis:reconnect_on_error:124] 
Retry limit exceeded while trying to reconnect to the Celery redis result store backend. The Celery application must be restarted.

ERROR [2022-06-21 05:05:44,213] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (0/20) now.
ERROR [2022-06-21 05:05:45,240] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (1/20) in 1.00 second.
ERROR [2022-06-21 05:05:47,263] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (2/20) in 1.00 second.
ERROR [2022-06-21 05:05:49,295] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (3/20) in 1.00 second.
ERROR [2022-06-21 05:05:51,347] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (4/20) in 1.00 second.
ERROR [2022-06-21 05:05:53,375] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (5/20) in 1.00 second.
ERROR [2022-06-21 05:05:55,397] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (6/20) in 1.00 second.
ERROR [2022-06-21 05:05:57,432] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (7/20) in 1.00 second.
ERROR [2022-06-21 05:05:59,458] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (8/20) in 1.00 second.
ERROR [2022-06-21 05:06:01,500] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (9/20) in 1.00 second.
ERROR [2022-06-21 05:06:03,523] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (10/20) in 1.00 second.
ERROR [2022-06-21 05:06:05,553] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (11/20) in 1.00 second.
ERROR [2022-06-21 05:06:07,577] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (12/20) in 1.00 second.
ERROR [2022-06-21 05:06:09,619] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (13/20) in 1.00 second.
ERROR [2022-06-21 05:06:11,643] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (14/20) in 1.00 second.
ERROR [2022-06-21 05:06:13,669] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (15/20) in 1.00 second.
ERROR [2022-06-21 05:06:15,691] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (16/20) in 1.00 second.
ERROR [2022-06-21 05:06:17,701] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (17/20) in 1.00 second.
ERROR [2022-06-21 05:06:19,733] - [celery.backends.redis:on_connection_error:383] Connection to Redis lost: Retry (18/20) in 1.00 second.
